loader/sky-pages-module/index.js

```diff
@@ -125,6 +125,10 @@ function join(items, sep) {
   return items.join(sep);
 }
 
+function importPath(path) {
```

Comment:
Is this method necessary?  Could instead we alter what pathWeb is produced as?  If it is necessary, would you mind documenting the method for consistency?